File: fixtures/conformance/php/src/Service.php
<?php
// Production service source. The text-search scenario looks for the needle
// comment below in exactly this file.
// CONFORMANCE_TEXT_NEEDLE: payment pipeline marker
namespace Conf;

use Conf\Util\FormatHelper as FormatHelperAlias;

/** Uniquely named caller of Conf::shared_unique_target. */
function dispatch_conformance_request(): void
{
    shared_unique_target();
    (new FormatHelperAlias())->format('dispatch');
}

File: fixtures/conformance/rust/src/service.rs
//! Production service module. The text-search scenario looks for the needle
//! comment below in exactly this file.

// CONFORMANCE_TEXT_NEEDLE: payment pipeline marker

use crate::shared::record_conformance_event as audit_event;
use crate::shared::shared_unique_target;

/// Uniquely named caller of `shared::shared_unique_target`.
pub fn dispatch_conformance_request() {
    shared_unique_target();
    audit_event("dispatch");
}
